consts/errors.js

```diff
@@ -56,12 +61,25 @@ const errors = {
   },
   EMAIL_NOT_FOUND: {
     code: 'EMAIL_NOT_FOUND',
-    message: 'There\'s no user registered with that email.'
+    message: "There's no user registered with that email."
```

Comment:
suggestion: 
try to avoid contractions like "there's" where you use singe quote coz prettier will transform it into what it is now and according to the rules we have to use single quotes. so 'there is' may be will be better?

Reply:
Yeah, got it, will modify